### PR TITLE
Document canal cdc

### PR DESCRIPTION
**Pull request by hengm3467**
## Info
- **Description**: 
Document the support for TiCDC (a dialect of Canal CDC)

- **Preview**: 
[ Paste the preview link to the edited page here. ]

- **Related code PR**: 
[ Provide the link to the code PR here. For example, https://github.com/risingwavelabs/risingwave/pull/4085. Delete this part if there's no code PR related. ]

- **Related doc issue**: 
Resolves [ Provide the link to the doc issue here. ]

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

### CODE REVIEW

File: docs/create-source/create-source-cdc.md [hunk 1]
```diff
@@ -5,21 +5,29 @@ description: Ingest CDC data via Kafka.
...
 
 RisingWave provides native MySQL and PostgreSQL CDC connectors. With these CDC connectors, you can ingest CDC data from these databases directly, without setting up additional services like Kafka.
 
-If Kafka is part of your technical stack, you can also use the Kafka connector in RisingWave to ingest CDC data in the form of Kafka topics from databases into RisingWave. You need to use a CDC tool such as [Debezium connector for MySQL](https://debezium.io/documentation/reference/stable/connectors/mysql.html) or [Maxwell's daemon](https://maxwells-daemon.io/) to convert CDC data into Kafka topics.
+If Kafka is part of your technical stack, you can also also use the Kafka connector in RisingWave to ingest CDC data in the form of Kafka topics from databases into RisingWave. You need to use a CDC tool such as [Debezium connector for MySQL](https://debezium.io/documentation/reference/stable/connectors/mysql.html), [Maxwell's daemon](https://maxwells-daemon.io/), or [TiCDC](https://docs.pingcap.com/tidb/dev/ticdc-overview) to convert CDC data into Kafka topics.
```
Comment (CharlieSYH) [inline docs/create-source/create-source-cdc.md]: Remove the extra "also" in the first sentence.

File: docs/data-ingestion.md [hunk 2]
```diff
@@ -9,22 +9,38 @@ You can ingest data into RisingWave in two ways:
 - Connect to and ingest data from external sources such as databases and message brokers.
 - Insert data to tables directly.
 
-## Ingest data from external data sources
+## Ingest data from sources
 
-A source is a resource that RisingWave can read data from. You can create a source in RisingWave using the `CREATE SOURCE` command. If you want to persist the data from the source, you need to create a table with connector settings using the `CREATE TABLE` command.
+A source is a resource that RisingWave can read data from. You can create a source in RisingWave using the [`CREATE SOURCE`](/sql/commands/sql-create-source.md)command. If you want to persist the data from the source, you need to create a table with connector settings using the [`CREATE TABLE`](/sql/commands/sql-create-source.md) command.
```
Comment (CharlieSYH) [inline docs/data-ingestion.md]: Missing an empty space before the first "command": "[`CREATE SOURCE`](/sql/commands/sql-create-source.md) command"

File: docs/data-ingestion.md [hunk 3]
```diff
@@ -36,10 +52,86 @@ If Kafka is part of your technical stack, you can also use the Kafka connector i
...
+
+When creating a source from streams in Maxwell JSON, you can define the schema of the source within the parentheses after the source name ((`schema_definition` in the syntax), and specify the format in the `ROW FORMAT` section. You can directly reference data fields in the JSON payload by their names as column names in the schema.
+
+Syntax:
+```sql
+ROW FORMAT MAXWELL
+```
+### Canal JSON
+
+RisingWave supports the TiCDC dialect of the Canal CDC format. When creating a source from streams in TiCDC, you can define the schema of the source within the parentheses after the source name ((`schema_definition` in the syntax), and specify the format in the `ROW FORMAT` section. You can directly reference data fields in the JSON payload by their names as column names in the schema.
```
Comment (CharlieSYH) [inline docs/data-ingestion.md]: Remove the extra "(" here:
((`schema_definition`
->
(`schema_definition`

File: versioned_docs/version-0.1.17/create-source/create-source-cdc.md [hunk 4]
```diff
@@ -5,22 +5,29 @@ description: Ingest CDC data via Kafka.
...
 
 RisingWave provides native MySQL and PostgreSQL CDC connectors. With these CDC connectors, you can ingest CDC data from these databases directly, without setting up additional services like Kafka.
 
-If Kafka is part of your technical stack, you can also use the Kafka connector in RisingWave to ingest CDC data in the form of Kafka topics from databases into RisingWave. You need to use a CDC tool such as [Debezium connector for MySQL](https://debezium.io/documentation/reference/stable/connectors/mysql.html) or [Maxwell's daemon](https://maxwells-daemon.io/) to convert CDC data into Kafka topics.
+If Kafka is part of your technical stack, you can also also use the Kafka connector in RisingWave to ingest CDC data in the form of Kafka topics from databases into RisingWave. You need to use a CDC tool such as [Debezium connector for MySQL](https://debezium.io/documentation/reference/stable/connectors/mysql.html), [Maxwell's daemon](https://maxwells-daemon.io/), or [TiCDC](https://docs.pingcap.com/tidb/dev/ticdc-overview) to convert CDC data into Kafka topics.
```
Comment (CharlieSYH) [inline versioned_docs/version-0.1.17/create-source/create-source-cdc.md]: Remove the extra "also"

**Review by tabVersion — APPROVED** [review]

**Comment by hengm3467** [comment]
Thanks for the comments! @CharlieSYH and @tabVersion

**Pull request merged**